### ISSUE TITLE
presponder is listed in README as supported on OS X, but fails due to UIResponder reference

### ISSUE THREAD

**Issue by mattreagan29** [facebook/chisel#134]
`presponder` is listed in the README as supported on OS X but using on OS X throws an error related to its reference to UIResponder in FBPrintCommands.py

Ex:

```
(lldb) p (BOOL)[self isKindOfClass:[NSResponder class]]
(BOOL) $3 = YES
(lldb) presponder self
error: error: use of undeclared identifier 'UIResponder'
error: 1 errors parsing expression
Traceback (most recent call last):
  File "/Users/matt/Documents/chisel-master/fblldb.py", line 79, in runCommand
    command.run(args, options)
  File "/Users/matt/Documents/chisel-master/commands/FBPrintCommands.py", line 179, in run
    if not fb.evaluateBooleanExpression('(BOOL)[(id)' + startResponder + ' isKindOfClass:[UIResponder class]]') and not fb.evaluateBooleanExpression('(BOOL)[(id)' + startResponder + ' isKindOfClass:[NSResponder class]]'):
  File "/Users/matt/Documents/chisel-master/fblldbbase.py", line 69, in evaluateBooleanExpression
    return (int(evaluateIntegerExpression('(BOOL)(' + expression + ')', printErrors)) != 0)
  File "/Users/matt/Documents/chisel-master/fblldbbase.py", line 61, in evaluateIntegerExpression
    output = evaluateExpression('(int)(' + expression + ')', printErrors).replace('\'', '')
AttributeError: 'NoneType' object has no attribute 'replace'
(lldb) 
```

Note: Merely changing or removing the UIResponder reference in `FBPrintCommands.py` allows the `presponder` call to work as expected

**Comment by kastiglione** [comment]
Thanks! Sounds like an easy fix. Pull requests happily accepted 😄

**Comment by kastiglione** [comment]
Fixed by @kolinkrewinkel in #135.
